indico/modules/rb_new/controllers/backend.py

```diff
@@ -236,3 +250,10 @@ class RHRoomSuggestions(RHRoomBookingBase):
     def _process(self, args):
         return jsonify([dict(suggestion, room=rooms_schema.dump(suggestion['room'], many=False).data)
                         for suggestion in get_suggestions(args, limit=NUM_SUGGESTIONS)])
+
+
+class RHRoomsSprite(RHRoomBookingBase):
+    def _process(self):
+        photo_data = _cache.get('rooms-sprite')
+        io = BytesIO(photo_data)
+        return send_file('rooms-sprite.jpg', io, 'image/jpeg')
```

Comment:
this should have strong caching headers (and a cache buster token in the url that's changed whenever the cache is invalidated)

Reply:
How do you know this token (e.g. a hash of the photo) from the client-side when fetching the url?